lib/litle/api.rb

```diff
@@ -185,6 +185,15 @@ def process_notification(notification, properties, context)
           # gw_notification.entity =
         end
       end
+
+      def set_payment_processor_account_id(currency, kb_tenant_id=nil)
+        config = ::Killbill::Plugin::ActiveMerchant.config(kb_tenant_id)
+        options = {}
+        if config[:multicurrency]
+          options = {:payment_processor_account_id => currency}
```

Comment:
It would be more flexible to have an indirect map, that ties each currency to the corresponding merchant account.

That way, we don't have to de-dup entries in the main config, for example:

```
:multicurrency:
  :USD: merchant_account_id_US
  :CAD: merchant_account_id_US
  :BRL: merchant_account_id_BRL
```

Make sure also to handle the missing config scenario, where a particular currency isn't configured (maybe fallback to the primary `:default` account?).